src/renderers/WebGLRenderer.js

```diff
@@ -1369,8 +1386,6 @@ class WebGLRenderer {
 
 			if ( _clippingEnabled === true ) clipping.setGlobalState( _this.clippingPlanes, camera );
 
-			if ( transmissiveObjects.length > 0 ) renderTransmissionPass( opaqueObjects, transmissiveObjects, scene, camera );
```

Comment:
After a final review I have one question. There is the line:
```js
if ( _clippingEnabled === true ) clipping.setGlobalState( _this.clippingPlanes, camera );
```
which was previously executed before `renderTransmissionPass()` was called. When this PR, global clipping planes are configured after the transmission pass which isn't right, imo. If you define global clipping planes, opaque objects might be clipped in the normal pass but not when seeing them through a transmissive object. So I think the `setGlobalState()` call needs to be moved as well.

Reply:
Since the call of `setGlobalState()` is camera-dependant, I would suggest to copy it into `renderTransmissionPass()`.

Reply:
You're right, it should be set beforehand. While I can see some use-case for only clipping in the normal pass, there's no reason for this PR to change the current behaviour.

Just moving this line won't be enough as the global clipping planes depend on the camera. For stereo rendering this means it'll have to be updated each time we switch eye. I'll replicating this line in `renderTransmissionPass()` as that should do the trick.

P.S. It seems global clipping planes are broken with nested render calls. Will open a issue for it